src/server_manager/web_app/app.ts

```diff
@@ -140,22 +154,31 @@ type DigitalOceanServerRepositoryFactory = (session: digitalocean_api.DigitalOce
 export class App {
   private digitalOceanRepository: server.ManagedServerRepository;
   private selectedServer: server.Server;
-  private serverBeingCreated: server.ManagedServer;
+  private idServer = new Map<string, server.Server>();
 
   constructor(
       private appRoot: AppRoot, private readonly version: string,
       private createDigitalOceanSession: DigitalOceanSessionFactory,
       private createDigitalOceanServerRepository: DigitalOceanServerRepositoryFactory,
       private manualServerRepository: server.ManualServerRepository,
-      private displayServerRepository: DisplayServerRepository,
       private digitalOceanTokenManager: TokenManager) {
     appRoot.setAttribute('outline-version', this.version);
 
-    appRoot.addEventListener('ConnectToDigitalOcean', (event: CustomEvent) => {
-      this.connectToDigitalOcean();
+    appRoot.addEventListener('ConnectDigitalOceanAccountRequested', (event: CustomEvent) => {
+      this.handleConnectDigitalOceanAccountRequest();
+    });
+    appRoot.addEventListener('CreateDigitalOceanServerRequested', (event: CustomEvent) => {
+      const accessToken = this.digitalOceanTokenManager?.getStoredToken();
```

Comment:
I like the optional chaining because it makes it clear that we don't need that object to be present. (e.g. you could pass null on a test). Less assumptions is good.

BTW, this code is going to change once we have a DigitalOceanAccount object. I don't think we need a "token manager" object.